server/controllers/transactions.js

```diff
@@ -1,12 +1,31 @@
 // import will go here
+import transcat from '../models/transactions';
 
 // the class for the transaction
 
-const transaction = {
-    // methodes will go here and you need to follow the same method
-     async save () {
-        console.log('the saving')
-    }
-}
+const Transaction = {
+  // methods will go here and you need to follow the same method
+  async save(req, res) {
+    const saved = req.body;
+    await transcat.saveMoney(saved)
+      .then((save) => {
+        if (!save) {
+          res.status(400).send({
+            status: 400,
```

Comment:
For avoiding forgetting to send the correct status code in the response JSON, I would suggest using res.statusCode on the status field in json.

Reply:
Thank you this is resolved